src/runtime/manifest.ts

```diff
@@ -985,6 +993,7 @@ ${e.message}
         connection.tags = connectionItem.target ? connectionItem.target.tags : [];
         const direction = {'->': 'out', '<-': 'in', '=': 'inout', 'consume': '`consume', 'provide': '`provide'}[connectionItem.dir];
```

Comment:
question for @shans backtick-consume (`enum) is not included in the default Direction enum.  Details?

Reply:
The \`consume and \`provide are still being incorporated. I'm working on it here: 
https://github.com/PolymerLabs/arcs/pull/3221